.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env sh
+set -eux
+
+TARGETOS=linux
+TARGETARCH=amd64
+
+# Install Kind
+curl -L -o kind "https://kind.sigs.k8s.io/dl/v0.22.0/kind-linux-${TARGETARCH}" && chmod +x ./kind
+sudo mv ./kind /usr/local/bin
+
+# Install kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/${TARGETARCH}/kubectl && chmod +x ./kubectl
+sudo mv ./kubectl /usr/local/bin
+
+# Install helm
```

Comment:
Do we need to install `helm` ? Because this is what I see when I ran `helm version` command on forked repo in the codespace.

```
sunnygoel87 ➜ /workspaces/idpbuilder (main) $ helm version
version.BuildInfo{Version:"v3.14.0", GitCommit:"3fc9f4b2638e76f26739cd77c7017139be81d0ea", GitTreeState:"clean", GoVersion:"go1.21.5"}
```

Reply:
@sunnygoel87 it depends what is the based image your using in codespaces.
I added a check to see if helm is not present and install it then. I'm not using codespaces I'm using [devpod](https://devpod.sh/)